conf/groups.conf.d/slurm.conf.example

```diff
@@ -3,19 +3,30 @@
 # Please see `man 5 groups.conf` for further details.
 #
 
+#
 # SLURM partition bindings
 #
-[slurm]
+[slurmpart,sp]
 map: sinfo -h -o "%N" -p $GROUP
 all: sinfo -h -o "%N"
 list: sinfo -h -o "%R"
 reverse: sinfo -h -N -o "%R" -n $NODE
 
+#
 # SLURM state bindings
 #
-[slurmstate]
-map: if [ "$GROUP" == "broken" ]; then sinfo -h -o "%N" -t down,drained; else sinfo -h -o "%N" -t $GROUP; fi
+[slurmstate,st]
+map: sinfo -h -o "%N" -t $GROUP
 all: sinfo -h -o "%N"
-list: sinfo -h -o "%T" | tr -d '~#$*'
-reverse: sinfo -h -N -o "%T" -n $NODE
-cache_time: 300
+list: sinfo -h -o "%T" | sed 's/[*$]//g'
```

Comment:
Could you explain why this sed is better? Do you have examples?

Reply:
Although the current regex doesn't use it, `sed` could allow to only remove the last character of the string if it matches the blacklist, while `tr` will blindly remove all occurrences. Granted, those characters don't really happen in node state names, but what if Slurm someday introduces a "SuperB0o$t" state, huh? :grin:

Performance wise, `sed` is anecdotally faster than `tr`, but the `tr` syntax is maybe easier to read.
So, for readability's sake, let's stick with `tr`. And add `@` and `+` to the list of characters to remove. From the `sinfo` man page:
```
NODE STATE CODES
       Node state codes are shortened as required for the field size.  These node states may be followed by a special character to identify state flags associated with the node.  The following node sufficies and states are used:
       *   The node is presently not responding and will not be allocated any new work.  If the node remains non-responsive, it will be placed in the DOWN state (except in the case of COMPLETING, DRAINED, DRAINING, FAIL, FAILING nodes).
       ~   The node is presently in a power saving mode (typically running at reduced frequency).
       #   The node is presently being powered up or configured.
       $   The node is currently in a reservation with a flag value of "maintenance".
       @   The node is pending reboot.
       ALLOCATED   The node has been allocated to one or more jobs.
       ALLOCATED+  The node is allocated to one or more active jobs plus one or more jobs are in the process of COMPLETING.
       COMPLETING [...]
```